### DISCUSSION TITLE
5120x1440

### DISCUSSION THREAD

**Discussion by ss7pro** [rxhanson/Rectangle#622] [Q&A]
Feature ideas are now kept in the [discussions tab](https://github.com/rxhanson/Rectangle/discussions). Please create an issue there, not here.

5120x1440 monitors are becoming more popular and this means more desk space. So far I was working on 4k (3840x2160) and I was perfectly fine with 3x3 l split. Any chance of getting 4x4 split added?

**Comment by rxhanson** [comment]
I don't have plans to add this to Rectangle, but it can be set up with the custom shortcut recorder in my paid superset of Rectangle, Hookshot (https://hookshot.app). Here's a post I wrote about it: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24
